package.json

```diff
@@ -3,6 +3,24 @@
   "version": "1.0.0",
   "description": "Unlock is a protocol for decentralized access control",
   "main": "index.js",
+  "private": true,
```

Comment:
Repo has to be marked private to enable use of workspaces.

Reply:
Does it have any other consequnce?

Reply:
I _believe_ it would only impact publishing this parent module on npm but I will double check.

Reply:
Confirmed, this would only prevent publishing the root-level unlock module. Does not interfere with subrepos.